CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.7.2
+
+* Fix  #188 subscribeToCharacteristic fails when characteristic config descriptor is not present.
+* Fix #195 scanFailure when using backgroundmodemessaging Firebase.
```

Comment:
Fix  the double space in the first line.

Use `code` quotation for identifiers.